heareval/tasks/util/audio.py

```diff
@@ -69,3 +71,12 @@ def resample_wav(in_file: str, out_file: str, out_sr: int):
     )
     # Make sure the return code is 0 and the command was successful.
     assert ret == 0
+
+
+def get_audiostats(in_file: str):
```

Comment:
Nice! We should include this in MonoWavTrimCorpus and only resample / convert to wav if the input needs it. If it doesn't then just create a symlink. Can do this in a separate PR.

Reply:
Yes. I was thinking of doing this. Will do.

Reply:
Awesome thanks